### PR TITLE
Changed type of property ToastActivatorCLSID in shortcut to VT_CLSID

### PR DESCRIPTION
**Pull request by dmitrya1979**
Currently ToastActivatorCLSID is VT_LPWSTR. As result toast notifications in Windows do not work since they expect VT_CLSID type. This minor fix corrects the issue

### CODE REVIEW

File: src/Squirrel/ShellFile.cs [hunk 1]
```diff
@@ -69,6 +69,45 @@ public static PropVariant FromString(string str)
 
                 return pv;
             }
+
+            public static PropVariant FromGuid(Guid guid)
+            {
+                byte[] bytes = ((Guid)guid).ToByteArray();
+                var pv = new PropVariant()
+                {
```
Comment (paulcbetts) [inline src/Squirrel/ShellFile.cs]: Can you follow the code style please.

File: src/Squirrel/ShellFile.cs [hunk 2]
```diff
@@ -885,11 +924,33 @@ public void SetAppUserModelId(string appId)
...
         {
             var propStore = (IPropertyStore)linkW;
+
             var pkey = PROPERTYKEY.PKEY_AppUserModel_ToastActivatorCLSID;
-            var str = PropVariant.FromString (clsid);
-            propStore.SetValue(ref pkey, ref str);
+
+            PropVariant varGuid = PropVariant.FromGuid(clsid);
+            try
+            {
```
Comment (paulcbetts) [inline src/Squirrel/ShellFile.cs]: Here too

File: src/Squirrel/ShellFile.cs [hunk 3]
```diff
@@ -885,11 +924,33 @@ public void SetAppUserModelId(string appId)
...
+                int errCode = propStore.SetValue(ref pkey, ref varGuid);
+                Marshal.ThrowExceptionForHR(errCode);
+
+                errCode = propStore.Commit();
+                Marshal.ThrowExceptionForHR(errCode);
+            }
+            finally
```
Comment (paulcbetts) [inline src/Squirrel/ShellFile.cs]: Here too

File: src/Squirrel/ShellFile.cs [hunk 4]
```diff
@@ -885,11 +924,33 @@ public void SetAppUserModelId(string appId)
         /// Sets the ToastActivatorCLSID
         /// </summary>
         public void SetToastActivatorCLSID(string clsid)
+        {
+            Guid guid = Guid.Parse(clsid);
```
Comment (paulcbetts) [inline src/Squirrel/ShellFile.cs]: var

**Comment by paulcbetts** [comment]
Thanks for the PR, need to fix a few things then we can merge this

**Comment by dmitrya1979** [comment]
Hello Paul, were my changes OK or I can help with something?

**Comment by paulcbetts** [comment]
@dmitrya1979 Thanks, sorry I missed this!

**Pull request merged**